src/test/resources/application.yml

```diff
@@ -44,6 +44,7 @@ govNotify:
 invite:
   url: ${INVITE_SIGNUP_URL:http://localhost:8080/signup/%s}
   validityInSeconds: ${INVITE_VALIDITY:259200}
+  allowedTimeForReReg: ${REREG_VALIDITY:86400}
```

Comment:
Please change the this property name to durationAfterReRegAllowedInSeconds

Reply:
Done